datadog/resource_datadog_downtime.go

```diff
@@ -108,18 +121,23 @@ func resourceDatadogDowntime() *schema.Resource {
 				},
 			},
 			"scope": {
-				Type:     schema.TypeList,
-				Required: true,
-				Elem:     &schema.Schema{Type: schema.TypeString},
+				Type:        schema.TypeList,
+				Required:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
+				Description: "specify the group scope to which this downtime applies. For everything use '*'",
 			},
 			"monitor_id": {
-				Type:     schema.TypeInt,
-				Optional: true,
+				Type:          schema.TypeInt,
+				Optional:      true,
+				ConflictsWith: []string{"monitor_tags"},
```

Comment:
Is setting both not an available option here? Would the API return an error previously if both were being sent?

Reply:
while technically it works, it's one of those, `it's broken if the tags also don't match the monitor and won't apply` it'll be an API validation in the future. The matching logic today is: (pseudo-python): `if (downtime.monitor_id is None or downtime.monitor_id == monitor.id) and monitor.tags.issuperset(downtime.monitor_tags)`

Reply:
Ok makes sense, thanks for the explanation :)